tests/jest-dom-matchers/toBeInTheDocument.test.ts

```diff
@@ -0,0 +1,10 @@
+import { createTab } from 'test-mule';
+
+test.skip('toBeInTheDocument', async () => {
```

Comment:
I fixed the bug on `main` that caused this matcher to not work, it should work now